sql/src/main/java/io/crate/operation/scalar/string/LowerFunction.java

```diff
@@ -68,6 +68,11 @@ public LowerFunction(FunctionInfo info) {
         this.info = info;
     }
 
+    public LowerFunction(FunctionInfo info, Locale currentLocale) {
+        this(info);
+        this.currentLocale = currentLocale;
```

Comment:
currentLocale cannot be final, because it is being assigned to further down in the code